packages/launchpad/src/checklist-item/style.scss

```diff
@@ -163,16 +165,19 @@
 .checklist-item__chevron {
 	display: flex;
 	line-height: 20px;
-	text-align: left;
 	width: 30px;
-	margin-left: auto;
 }
 
 .gridicon.checklist-item__chevron {
-	top: 0;
+	top: 2px;
 	margin-top: 0;
 }
 
+.checklist-item__counter {
+	margin-right: 5px;
```

Comment:
Looking at the screenshot in the original issue, we need a bit more margin on the right. It looks closer to `1em`/`14px` as far as I can tell.